### ISSUE TITLE
-o breaks on empty directories

### ISSUE THREAD

**Issue by deadok22** [gojuno/minimock#28]
Specifying an empty output directory for `-o` option results in `minimock` failing.

Steps to reproduce:
```
$ mkdir mocks
$ go run github.com/gojuno/minimock/cmd/minimock -g -i io.Reader -o mocks/ -s _mock.go
minimock: failed to load destination package: ./mocks: -: unknown import path "your/module/pkg/mocks": cannot find module providing package your/module/pkg/mocks
exit status 1
```

**Comment by hexdigest** [comment]
@deadok22 this should be fixed in 2.1.4, please check

**Comment by deadok22** [comment]
Awesome! Thank you @hexdigest!
